python/functional-programming/arrays-ii/the-filter-built-in-function.md

```diff
@@ -104,15 +104,26 @@ What is the printed result of the following code execution?
 numbers = [-3, -2, -1, 0, 1, 2, 3]
 def mystery_function(element):
   return element < 0
-print(filter(mystery_function, numbers))
```

Comment:
Could have just converted the filter object to list,
```py
filtered = filter(mystery_function, numbers)
print(list(filtered))
```

I think, this approach is straightforward instead of the for loop..